salt/beacons/__init__.py

```diff
@@ -214,6 +214,45 @@ def list_beacons(self):
 
         return True
 
+    def list_available_beacons(self):
+        '''
+        List the available beacons
+        '''
+        _beacons = ['{0}'.format(_beacon.replace('.beacon', ''))
+                    for _beacon in list(self.beacons) if '.beacon' in _beacon]
```

Comment:
The call to `list()` here should not be necessary, iterating over a dict (or a lazy dict in this case) will iterate over its keys, just as calling `list(somedict)` will give you a list of its keys.